test/unit/core/attestations-registry/attestations-registry-config-logic.test.ts

```diff
@@ -531,6 +537,189 @@ describe('Test Attestations Registry Config Logic contract', () => {
+      NOT_CREATED: 50,
+    };
+
+    before(async () => {
+      await evmRevert(hre, snapshotId);
+    });
+
+    describe('Tag creation', async () => {
+      it('Should revert when creating a new tag as a non-owner', async () => {
+        await expect(
+          attestationsRegistry
+            .connect(notOwner)
+            .createNewTag(TAGS.TEST_INSERTION, formatBytes32String('TEST_INSERTION'))
+        ).to.be.revertedWith('Ownable: caller is not the owner');
+      });
+
+      it('Should create a new tag as an owner', async () => {
+        const tagInserted = await attestationsRegistry
+          .connect(deployer)
+          .createNewTag(TAGS.TEST_INSERTION, formatBytes32String('TEST_INSERTION'));
+
+        await expect(tagInserted)
+          .to.emit(attestationsRegistry, 'NewTagInserted')
+          .withArgs(TAGS.TEST_INSERTION, formatBytes32String('TEST_INSERTION'));
+      });
+
+      it('Should revert when creating a tag with index > 63', async () => {
+        await expect(
+          attestationsRegistry.createNewTag(64, formatBytes32String('TAG_OVERFLOW'))
+        ).to.be.revertedWith('TagIndexOutOfBounds(64)');
+      });
+
+      it('Should revert when trying to re-creating a tag for the same index', async () => {
+        await expect(
+          attestationsRegistry.createNewTag(TAGS.TEST_INSERTION, formatBytes32String('OTHER TAG'))
+        ).to.be.revertedWith('TagAlreadyExists(10)');
+      });
+    });
+
+    describe('Tag update', async () => {
+      it('Should revert when updating tag as a non-owner', async () => {
+        await expect(
+          attestationsRegistry
+            .connect(notOwner)
+            .updateTagName(TAGS.TEST_INSERTION, formatBytes32String('CURATED2'))
+        ).to.be.revertedWith('Ownable: caller is not the owner');
+      });
+
+      it('Should revert when trying to update update a tag name that does not exists', async () => {
+        await expect(
+          attestationsRegistry
+            .connect(deployer)
+            .updateTagName(TAGS.NOT_CREATED, formatBytes32String('NOT_INSERTED'))
+        ).to.be.revertedWith('TagDoesNotExist(50)');
+      });
+
+      it('Should update a tag name', async () => {
+        const tagUpdated = await attestationsRegistry
+          .connect(deployer)
+          .updateTagName(TAGS.TEST_INSERTION, formatBytes32String('CURATED2'));
+
+        await expect(tagUpdated)
+          .to.emit(attestationsRegistry, 'TagNameUpdated')
+          .withArgs(TAGS.TEST_INSERTION, formatBytes32String('CURATED2'));
+      });
+    });
+
+    describe('Tag deletion', async () => {
+      it('Should revert when deleting a tag as a non-owner', async () => {
+        await expect(
+          attestationsRegistry.connect(notOwner).deleteTag(TAGS.NOT_CREATED)
+        ).to.be.revertedWith('Ownable: caller is not the owner');
+      });
+
+      it('Should revert when trying to update update a tag name that does not exists', async () => {
```

Comment:
typo "when trying to delete" instead "trying to update" + wrong test -> the error message should be `TagDoesNotExist(50)`. Should connect with `deployer` instead of `notOwner`